handler/src/main/java/io/netty5/handler/ssl/SslContextBuilder.java

```diff
@@ -608,10 +647,14 @@ public SslContext build() throws SSLException {
                 ciphers, cipherFilter, apn, sessionCacheSize, sessionTimeout, clientAuth, protocols, startTls,
                 enableOcsp, keyStoreType, toArray(options.entrySet(), EMPTY_ENTRIES));
         } else {
+            String endpointIA = null;
+            if (!PlatformDependent.isAndroid() || PlatformDependent.androidApiVersion() >= 24) {
+                endpointIA = endpointIdentificationAlgorithm;
```

Comment:
What happens on older Android if we passed an EIA value?

Reply:
To be honest I haven't checked, but I think that something like NoSuchMethodError because this method was added since 24 version
https://developer.android.com/reference/javax/net/ssl/SSLParameters#setEndpointIdentificationAlgorithm(java.lang.String)

Maybe it is better to throw IllegalStateException from SslContextBuilder methods in this case
```java
endpointIdentificationAlgorithm(String value){
  if(value != null && PlatformDependent.isAndroid() && PlatformDependent.androidApiVersion() < 24){
     throw new IllegalStateException(...);
  }
}
```

Reply:
fixed